data/json/effects.json

```diff
@@ -362,6 +367,22 @@
     },
     {
         "type": "effect_type",
+        "id": "numbpoison",
+        "name": ["Numbness"],
+        "desc": ["You feel sluggishness and numbness spreading around your body!"],
+        "miss_messages": [["You feel numb.", 1]],
+        "rating": "mixed",
+        "resist_traits": ["POISRESIST"],
+        "main_parts_only": true,
+        "base_mods": {
+            "dex_mod": [-3, -1],
+            "speed_mod": [-15, -5],
+            "pkill_tick": [5],
+            "pkill_min": [1]
```

Comment:
Should probably have a max value.
The `pkill` effect will keep ticking, while dexterity and speed stays constantly debuffed by the same value.
This could even lead to unexpected player death, as huge pkill values (achievable with 2 doses of heroin in current game) cause instakill.

Plus, it's a bit too positive with uncapped pkill. A poison should be a poison first and only a medicine when applied in low, controlled doses.